src/d3a/models/strategy/storage.py

```diff
@@ -262,11 +262,12 @@ def event_tick(self):
         self.state.clamp_energy_to_buy_kWh([ma.time_slot for ma in self.area.all_markets])
         for market in self.area.all_markets:
             if ConstSettings.IAASettings.MARKET_TYPE == 1:
-                if self.bid_update.get_price_update_point(self, market.time_slot):
-                    self.buy_energy(market)
+                self.bid_update.increment_update_counter(self, market.time_slot)
+                self.buy_energy(market)
```

Comment:
Nice, it could also be used for LoadStrategy too so that it can also be able to change its buying preference the over the elapsed time. May be not now but for future improvement.

Reply:
So you mean to separate `increment_update_counter` from `update_posted_bids_over_ticks` ?
Yes, sure. I also have to do it for `update_offer` then. I dont know why id didnt do it in the first place.

Reply:
(all functions I named are in `UpdateFrequencyMixin` )

Reply:
Sorry, I wasn't clear with my first comment. What I meant was `LoadStrategy` wasn't able to update its buying_rate for **single-sided-market** over the elapsed time. But using `increment_update_counter ` could be helpful in updating its buying preference after every update_interval.

Reply:
OMG! yes! Why isn't this there already. This is a bug IMO. 
@spyrostz shall I add a separate story for this or rather fix it here. IMO the first as this story already remains 3 weeks in "in progress".

Reply:
This is a bug IMO too. @hannesdiedrich feel free to open a separate story for that.